### PR TITLE
New Keyboard: SiddersKB Majbritt (Pronounced My Brit)

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

New Keyboard - Alice Style with arrows. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sidderskb/majbritt/majbritt.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    // runs for every action, just before processing by the firmware
+
+    return process_record_user(keycode, record);
+}
+
+void led_set_kb(uint8_t usb_led) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+    led_set_user(usb_led);
+}
```
Comment (drashna) [inline keyboards/sidderskb/majbritt/majbritt.c]: All of these are using the default config, so they can be safely removed. 

```suggestion
```

File: keyboards/sidderskb/majbritt/rules.mk [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (fauxpark) [inline keyboards/sidderskb/majbritt/rules.mk]:
```suggestion
```

File: keyboards/sidderskb/majbritt/keymaps/default/config.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
```
Comment (fauxpark) [inline keyboards/sidderskb/majbritt/keymaps/default/config.h]:
```suggestion
```

File: keyboards/sidderskb/majbritt/keymaps/default/config.h [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (fauxpark) [inline keyboards/sidderskb/majbritt/keymaps/default/config.h]: This file can be deleted, completely.

File: keyboards/sidderskb/majbritt/keymaps/via/rules.mk [hunk 5]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```
Comment (Duckle29) [inline keyboards/sidderskb/majbritt/keymaps/via/rules.mk]: Any particular reason LTO is only enabled for VIA?

**Pull request merged**